framework/db/transaction_manager.py

```diff
@@ -438,6 +440,44 @@ def GetByIDAsDict(self, trans_id, target_id=None):
         return self.DeriveTransactionDict(transaction_obj, include_raw_data=True)
 
     @target_required
+    def GetHrtResponse(self, filter_data, trans_id, target_id=None):
+        transaction_obj = self.db.session.query(models.Transaction).filter_by(target_id=target_id, id=trans_id).first()
+
+        if 'language' in filter_data:
+            # Validate the request. It should contain language as POST parameter.
+            language = filter_data['language']
+        else:
+            response = "Please specify the language."
+            return response
+
+        # If target not found. Raise error.
+        if not transaction_obj:
+            raise InvalidTransactionReference("No transaction with %s exists" % str(trans_id))
+
+        transaction_obj_dict = dict(transaction_obj.__dict__)
+        raw_request = transaction_obj_dict['raw_request']
+        method = transaction_obj_dict['method']
+
+        # Due to issue https://github.com/owtf/http-request-translator/issues/53
+        if method not in ['GET', 'POST']:
+            response = "Request cannnot be completed.\nOnly GET and POST method supported."
+            return response
+
+        try:
+            hrt_obj = HttpRequestTranslator(
+                request=raw_request,
+                languages=language,
+                proxy=None,
```

Comment:
The user should be able to use these parameters. It would require additional UI input to allow the user specifying these.